windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="creating-windows-machineset-gcp"]
+= Creating a Windows `MachineSet` object on GCP
+include::_attributes/common-attributes.adoc[]
+:context: creating-windows-machineset-gcp
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on Google Cloud Platform (GCP). For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
+
+[discrete]
+== Prerequisites
+
+* You installed the Windows Machine Config Operator (WMCO) using Operator Lifecycle Manager (OLM).
+* You are using a supported Windows Server as the operating system image. 
+
+include::modules/machine-api-overview.adoc[leveloffset=+1]
+include::modules/windows-machineset-gcp.adoc[leveloffset=+1]
+include::modules/machineset-creating.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
+
+* For more information on managing machine sets, see xref:../../machine_management/index.adoc#overview-of-machine-management[Overview of machine management].
```

Comment:
This link in the preview gives me a 404 error

Reply:
@jldohmann All apologies. I sent only the Windows Containers docs to file.rdu.redhat to save space. The Netlify preview works: https://49719--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.html#additional-resources

Reply:
No worries, thanks for directing me to the netlify preview to test the link!